Dockerfile

```diff
@@ -26,7 +37,7 @@ ENV HK_INFO_DELETE_HRS=12
 LABEL maintainer="Nick Satterly <nick.satterly@gmail.com>"
 LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.url="https://alerta.io" \
-      org.label-schema.vcs-url="https://github.com/alerta/docker-alerta" \
+      org.label-schema.vcs-url="https://github.com/g-research/docker-alerta" \
       org.label-schema.vcs-ref=$VCS_REF \
       org.label-schema.version=$VERSION \
       org.label-schema.schema-version="1.0.0-rc.1"
```

Comment:
I think we should just set the labels with `docker/metadata-action` instead, and then we can even drop the `BUILD_DATE` and `VCS_REF` variables